website/events/2021-06-15-release-apache-apisix-dashboard-2.7.md

```diff
@@ -1,5 +1,22 @@
 ---
 title: Release Apache APISIX Dashboard 2.7
+keywords:
+- apisix
```

Comment:
I don't think we need too many duplicated keywords, Search Engine will take them as the same keyword AFAIK 🤔

Reply:
so we can keep 'apisix' and delete others such as "APISIX"?

Reply:
deleted duplicated keywords 'apisix' and 'apache apisix'